docs/user/metrics/features.csv

```diff
@@ -18,3 +18,4 @@ Library,ML Backend,I/O Backend,Spatial Backend,Transform Backend,Datasets,Weight
 `GeoTessera`_,scikit-learn,GDAL,-,geopandas,0,0,✅,❌,✅,❌,❌
 `OTBTF`_,TensorFlow,"GDAL, OTB",-,-,0,0,✅,❌,✅,❌,❌
 `GeoDeep`_,ONNX,GDAL,-,-,0,7,✅,❌,✅,❌,❌
+`torchange`_,PyTorch,"Apache Arrow, skimage",geopandas,Albumentations,8,8,❌,❌,❌,❌,✅
```

Comment:
* spatial backend: I don't see geopandas in the codebase. Perhaps there is no spatial backend and geographic metadata isn't used?
* weights: Where do I find your pretrained model weights?
* time series: Do you support variable length sequences of SITS data or only binary temporal pairs?